### PR TITLE
update 1.18 in readme

### PR DESCRIPTION
**Pull request by Bluscream**

### CODE REVIEW

**Comment by Boobies** [comment]
Seems like a pretty useless PR; to update the readme of this branch to mention "hey, check out the other branches".

File: README.md [hunk 1]
```diff
@@ -50,15 +51,15 @@ A Minecraft pathfinder bot.
...
 
 For 1.16.5, [click here](https://www.youtube.com/watch?v=_4eVJ9Qz2J8) and see description. If you need Forge or Fabric 1.16.5, look [here](https://github.com/cabaletta/baritone/releases/tag/v1.6.3) and get the `api-forge` or `api-fabric` jar. **For 1.16.5 Fabric, just click [here](https://github.com/cabaletta/baritone/releases/download/v1.6.3/baritone-api-fabric-1.6.3.jar)**.
 
 For 1.17.1 leijurv is to lazy to actually create a release so you can login to github and download the jars in the `artifacts` zip [here](https://github.com/cabaletta/baritone/actions/runs/1451572684)
 
-For 1.18.1 leijurv is to lazy to actually create a release so you can login to github and download the jars in the `artifacts` zip [here](https://github.com/cabaletta/baritone/actions/runs/1472825778)
+For 1.18.1 leijurv is to lazy to actually create a release so you can login to github and download the jars in the `artifacts` zip [here](https://github.com/cabaletta/baritone/suites/4536126920/artifacts/121555622) or check the runs for the 1.18 branch [here](https://github.com/cabaletta/baritone/actions/workflows/gradle_build.yml?query=branch%3A1.18)
```
Comment (wagyourtail) [inline README.md:62]: since not merged yet, could update to `https://github.com/cabaletta/baritone/suites/4890997528/artifacts/140232385`

**Comment by wagyourtail** [comment]
LGTM

**Comment by scorbett123** [comment]
Leijurv has finally stopped being lazy, so this pull request is no longer needed

**Comment by wagyourtail** [comment]
> Leijurv has finally stopped being lazy

for now

**Pull request closed without merging**